script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -141,7 +141,7 @@ Exactly who is that woman to you, Nii-san?"
 [sha:7ec8ec0efa509c3cd81a69ecf57281335ceabd0e]{
```

Comment:
On line 14 on this file I would like to change
'They were stern words, ones that I couldn't disobey.' -> 'They are stern words, ones that I can't disobey.'

Reply:
I kinda disagree with this one, the words have already been uttered.

Reply:
If I'm not having a massive smooth brain moment the tense of the sentence is defined by the verb. Since the verb is 'couldn't disobey' and he hasn't taken action yet, he still 'can't disobey' the words.

Reply:
I think you've hit on the crux, which is that "were/couldn't" reads to me as past/past (he's already accepted the fact he is unable to disobey). "are/cannot" reads as present/present, so that sounds fine. I'd either leave as is or change both so that the sentence is 
> They are stern words, ones that I cannot disobey.

Reply:
Aye, that's perfect.